src/aotcompile.cpp

```diff
@@ -373,30 +391,63 @@ void *jl_create_native_impl(jl_array_t *methods, LLVMOrcThreadSafeModuleRef llvm
                 continue;
             }
             mi = (jl_method_instance_t*)item;
+            compile_mi:
```

Comment:
```suggestion
compile_mi:
```

Or indent the rest? The label was difficult to spot the first go-around, and I agree we should refactor the code to avoid the goto.